tests/server/tso/global_tso_test.go

```diff
@@ -326,10 +330,9 @@ func (s *testTimeFallBackSuite) testGetTimestamp(c *C, n uint32) *pdpb.Timestamp
 	resp, err := tsoClient.Recv()
 	c.Assert(err, IsNil)
 	c.Assert(resp.GetCount(), Equals, uint32(n))
-
 	res := resp.GetTimestamp()
-	c.Assert(tsoutil.CompareTimestamp(res, tsoutil.GenerateTimestamp(time.Now(), 0)), Equals, 1)
-
+	c.Assert(res.GetPhysical(), Greater, int64(0))
```

Comment:
Why do we need to change it?

Reply:
The original code compares `res` with zero. In addition to this, the new codes will also check whether the logical part is bigger than `count`.

Reply:
It seems physical is not zero before?

Reply:
Oops, my mistake. Fixed.